### PR TITLE
Feature - Changed form overlay color to light

### PR DESCRIPTION
**Pull request by MoiyadKayda**
## Description

The color is not backgroundClear50 but I think this one looks better.

## Screenshots:
Screenshot (263)
image

## Checklist:

- [ ] I have updated the CHANGELOG with a summary of my changes
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: src/molecules/form/component.tsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ const LoadingOverlay = styled.div`
   position: absolute;
   top: 0;
   left: 0;
-  background-color: rgba(0, 0, 0, 0.7);
+  background-color: rgba(0, 0, 0, 0.05);
```
Comment (krishan711) [inline src/molecules/form/component.tsx:28]: this should not be black it should use the background color

Comment (MoiyadKayda) [reply src/molecules/form/component.tsx:28]: like it should be customization from the theme

Comment (krishan711) [reply src/molecules/form/component.tsx:28]: well it can either be added to the form theme or the background color from the theme.colors

Comment (krishan711) [reply src/molecules/form/component.tsx:28]: probably better is to add a overlayColor to the form theme and then set this to have a sensible default

**Pull request closed without merging**